install/kubernetes/templates/istio-auth/istio-egress-auth.yaml

```diff
@@ -1,50 +0,0 @@
-################################
```

Comment:
is auth/no-auth part of the variables/substitution and that's how you are able to delete all the -auth template ? (is it all or just those, if just those, how come?)